AIDevGallery/Samples/SharedCode/BitmapFunctions.cs

```diff
@@ -220,8 +216,11 @@ public static BitmapImage RenderPredictions(Bitmap image, List<Prediction> predi
             g.DrawLine(pen, p.Box.Xmax, p.Box.Ymax, p.Box.Xmin, p.Box.Ymax);
             g.DrawLine(pen, p.Box.Xmin, p.Box.Ymax, p.Box.Xmin, p.Box.Ymin);
 
-            // Draw the label and confidence
+            // Calculate adjusted font and  draw the label
+            float maxTextWidth = p.Box.Xmax - p.Box.Xmin;
             string labelText = $"{p.Label}, {p.Confidence:0.00}";
+            float adjustedFontSize = Math.Clamp(maxTextWidth / ((float)labelText.Length), 8, 16);
+            using Font font = new("Arial", adjustedFontSize);
```

Comment:
I would move definition outside of the loop, so we create only one reference, if possible.
I see the logic using the value from the specific prediction, but its already clamped between 8 and 16. Couldn't we pick a value, or maybe the average between all predictions? Something like that. For example, select the team.jpg file we have on the gallery Assets. It renders each prediction with a slightly different size, which feels weird.